nautobot/cloud/models.py

```diff
@@ -7,7 +7,11 @@
 from nautobot.core.constants import CHARFIELD_MAX_LENGTH
 from nautobot.core.models import BaseModel
 from nautobot.core.models.generics import PrimaryModel
-from nautobot.extras.utils import extras_features
+from nautobot.extras.utils import CloudTypeModelsQuery, extras_features
+
+
+class CloudTypeCompatibleMixin:
```

Comment:
~~How does this mixin work in practice? I am confused since there is no class attribute or anything.~~